src/Sylius/Component/User/Repository/UserRepositoryInterface.php

```diff
@@ -15,18 +15,14 @@
 use Sylius\Component\User\Model\UserInterface;
 
 /**
- * User repository interface.
- *
  * @author Michał Marcinkowski <michal.marcinkowski@lakion.com>
  */
 interface UserRepositoryInterface extends RepositoryInterface
 {
     /**
-     * Finds user by email
-     *
      * @param string $email
      *
-     * @return null|UserInterface User object or null
+     * @return null|UserInterface
```

Comment:
Usually `null` is at the end of possible options, the expected behaviour is to get `UserInterface` or `null`: `@var UserInterface|null`